apps/molgenis-components/src/components/filters/FilterInput.vue

```diff
@@ -97,12 +125,26 @@ export default {
       <demo-item>
         <FilterInput
             id="filter-input-example-2"
+            columnType="STRING"
             :conditions="conditions1"
             @updateConditions="conditions1 = $event"
         ></FilterInput>
```

Comment:
nitpicking: since nothing is slotted we could just end the tag directly

Reply:
not a fan of self closing tags myself , but will update